sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/batch_deployment.py

```diff
@@ -34,52 +34,52 @@
 class BatchDeployment(Deployment):  # pylint: disable=too-many-instance-attributes
```

Comment:
Please add examples to all of the docstrings for classes and standalone methods (no need to add for attributes/properties). You can use this PR as an example:

https://github.com/Azure/azure-sdk-for-python/pull/32284/files

Also, here's azure-sdk-for-python's sample guide: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/sample_guide.md#capturing-code-snippets-in-reference-documentation